README.md

```diff
@@ -38,4 +38,45 @@ It includes a `MetadataInterceptor` that adds the `recipient` as grpc metadata t
 
 ## Versioning
 This project uses semantic versioning and releases a new version regularly upon adding support of a newer release of the camino messenger protocol.
-Version names do not have to match identically with the camino messenger protocol version, but they should be in sync.
\ No newline at end of file
+Version names do not have to match identically with the camino messenger protocol version, but they should be in sync.
+
+
+## Dependencies
+
+This chapter outlines the key dependencies required to build, package, and run the project. Ensure all dependencies are installed and properly configured before proceeding with the project setup.
```

Comment:
you don't need to install dependencies here, only maven and jre need to be installed, but the others they are written in pom.xml and you don't need to install them on your local machine

Reply:
changed